### PR TITLE
Add support for adding devices to container

### PR DESCRIPTION
**Pull request by rhatdan**
Also add --quiet option to kpod create/run since
this will help with writing tests.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

**Comment by mheon** [comment]
LGTM

File: docs/podman-create.1.md [hunk 1]
```diff
@@ -373,6 +373,10 @@ port to a random port on the host within an *ephemeral port range* defined by
 `/proc/sys/net/ipv4/ip_local_port_range`. To find the mapping between the host
 ports and the exposed ports, use `podman port`.
 
+**--quiet, -q**
+
+Suppress output information when pulling images
```
Comment (TomSweeneyRedHat) [inline docs/podman-create.1.md]: nit, period at end here.  But that being said, there are a lot of lines not ending in a period.  FWIW, I think all lines should end in a period, comments or not, but especially man pages or other customer facing output.  I might take a hack through the pages next week and try to tighten that up.  Yeah my OCD is kicking in.

File: docs/podman-run.1.md [hunk 2]
```diff
@@ -379,6 +379,10 @@ port to a random port on the host within an *ephemeral port range* defined by
 `/proc/sys/net/ipv4/ip_local_port_range`. To find the mapping between the host
 ports and the exposed ports, use `podman port`.
 
+**--quiet, -q**
+
+Suppress output information when pulling images
```
Comment (TomSweeneyRedHat) [inline docs/podman-run.1.md]: Ditto period.

File: docs/podman-create.1.md (hunk 1 above)
Comment (rhatdan) [reply docs/podman-create.1.md]: Lets merge this and take care of man pages in different PR.

**Comment by rhatdan** [comment]
@rh-atomic-bot r=TomSweeneyRedHat

**Pull request closed without merging**